src/autogluon/bench/eval/evaluation/preprocess/preprocess_openml.py

```diff
@@ -58,7 +59,7 @@ def preprocess_openml_input(
 
     cleaned_input = preprocess_utils.clean_result(raw_input, folds_to_keep=folds_to_keep, remove_invalid=False)
 
-    cleaned_input["tid"] = [int(x.split("/")[-1]) for x in cleaned_input["id"]]
+    cleaned_input["tid"] = [int(part) for x in cleaned_input["id"] for part in re.split(r'[/_]', x)[-1:] if part.isdigit()]
 
```

Comment:
Does this code assume that every timeseries task name ends with a suffix `_n`, where `n` is a unique integer for each task?

If that's the case, could you add a note to the example time series task definition so that we can keep track of this requirement?

Reply:
Yes, as per the last discussion we had decided to put a unique integer for each task.
I will add this note in the `benchmark.yaml` file for timeseries.